pkg/quota/evaluator/core/persistent_volume_claims.go

```diff
@@ -147,13 +150,12 @@ func (p *pvcEvaluator) Handles(a admission.Attributes) bool {
 	if op == admission.Create {
 		return true
 	}
-	updateUninitialized, err := util.IsUpdatingUninitializedObject(a)
+	initializationCompletion, err := util.IsInitializationCompletion(a)
```

Comment:
Should the below error not be handled or logged in some way?

Reply:
How about runtime.HandleError()? Is it ok to invoke it from pkg/quota?

Reply:
sure, and I think so. There is another instance below.